test/bridge.test.ts

```diff
@@ -486,24 +477,34 @@ describe("Bridge", async function () {
     // check last update of L2 tokens
     expect(await l2StaticADai.call('get_last_update', {})).to.deep.equal({ block_number: {high: 0n, low: BigInt(blockNumberDai) }});
     expect(await l2StaticAUsdc.call('get_last_update', {})).to.deep.equal({ block_number: {high: 0n, low: BigInt(blockNumberUsdc) }});
-
-    // check balance of L2 tokens
-    expect(await l2StaticADai.call('balanceOf', { account: BigInt(l2user.starknetContract.address) })).to.deep.equal({ balance: { high: 0n, low:  0n } });
-    expect(await l2StaticAUsdc.call('balanceOf', { account: BigInt(l2user.starknetContract.address) })).to.deep.equal({ balance: { high: 0n, low:  0n } });
   })
 
   it('L2 user sends back reward accrued to L1 user', async () => {
-    ////// Needs to be implemented when token bridge is updated
-   // // flush L2 messages to be consumed by L1
-   //  const flushL2Response = await starknet.devnet.flush();
-   //  const flushL2Messages = flushL2Response.consumed_messages.from_l2;
-   //  expect(flushL2Response.consumed_messages.from_l1).to.be.empty;
-   //  expect(flushL2Messages).to.have.a.lengthOf(1);
-
-   //  // call recieveRewards on L1 to consume messages from L2
-   //  await l1Bridge.connect(l1user).receiveRewards(l1StaticDai.address, l1user.address, 30);
-
-   //  // check that the l1 user received reward tokens
-   //  expect(await l1AAVE.balanceOf(l1user.address)).to.be.equal(30);
+    // check claimable rewards
+    const claimableADai = await l2StaticADai.call('get_user_claimable_rewards', {user: BigInt(l2user.starknetContract.address)});
+    const claimableAUsdc = await l2StaticAUsdc.call('get_user_claimable_rewards', {user: BigInt(l2user.starknetContract.address)});
+
+    // claim rewards on L2
+    await l2user.invoke(l2StaticADai, "claim_rewards", { recipient: BigInt(l2user.starknetContract.address) });
+    await l2user.invoke(l2StaticAUsdc, "claim_rewards", { recipient: BigInt(l2user.starknetContract.address) });
+    const claimed = await l2rewAAVE.call("balanceOf", { account: BigInt(l2user.starknetContract.address) });
+    expect(claimed).to.deep.equal({balance: {low: claimableADai["user_claimable_rewards"]["low"] + claimableAUsdc["user_claimable_rewards"]["low"], high: 0n}});
+
+    // bridge rewards
+    const rewardsAmountBridged = 1000n; 
+    await l2user.invoke(l2Bridge, "bridge_rewards", { l1_recipient: BigInt(l1user.address), amount: { high: 0n, low: rewardsAmountBridged } });
```

Comment:
Why are we using `rewardsAmountBridged` instead of the recently `claimed` rewAAVE amount?

Reply:
done